include/rocksdb/attribute_groups.h

```diff
@@ -0,0 +1,104 @@
+//  Copyright (c) Meta Platforms, Inc. and affiliates.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#pragma once
+
+#include <ostream>
+#include <tuple>
+#include <utility>
```

Comment:
Some/all of these might be unnecessary. Also potentially some of the RocksDB includes below like `slice.h` or `comparator.h`